src/main/java/org/sasanlabs/internal/utility/GenericUtils.java

```diff
@@ -9,11 +9,14 @@ public class GenericUtils {
 
     public static final String LOCALHOST = "127.0.0.1";
 
+    private GenericUtils() {}
```

Comment:
can you please mark this class as final as well?

Reply:
sure